MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include *.txt README.rst CHANGELOG .install.sh version.py h3_version.py
+global-exclude *.pyc
```

Comment:
No newline at EOF. `cat` won't work nicely here.

Reply:
add the newline. thanks